app/assets/javascripts/spree/frontend/spree_payu_integration.js

```diff
@@ -1 +1,31 @@
 /* Placeholder for backend dummy app */
+SpreePayuIntegration = {
+  updateSaveAndContinueVisibility: function() {
+    if (this.isButtonHidden()) {
+      $(this).trigger('hideSaveAndContinue')
```

Comment:
why not a regular JS call and you are wrapping this object in `$(...)` ? 

Why not `this.hideSaveAndContinue();`